TestRunner/Program.cs

```diff
@@ -44,45 +44,47 @@ static void Main(string[] args)
         {
             LoadAllTestAssemblies();
 
-            if (args.Length == 0)
+
+            while (args.Length == 0 || !m_TestMethods.ContainsKey(args[0]))
             {
-                Console.WriteLine("Please provide a filter for the methods you want to run. This can be either the name of the method or its namespace (after 'BH.Test')");
-                return;
+                if (args.Length == 0)
+                    Console.WriteLine("Please provide a filter for the methods you want to run. This can be either the name of the method or its namespace (after 'BH.Test')");
+                else
+                    Console.WriteLine("Cannot find any test matching " + args[0]);
+
+                Console.WriteLine($"Avilable methods to run are: {string.Join(", ", m_TestMethods.Keys)}");
+
+                args = Console.ReadLine().Split(' ');
```

Comment:
Shouldn't this be simply:
```suggestion
                args[0] = Console.ReadLine();
```

...otherwise you may lose all other `args` than the first

Reply:
think we should split to allow for more than one argument, like for verisoning check, you can pass a boolean to make it run all verisons or jsut oone